deploy/006_deploy_uma_emp_creator.js

```diff
@@ -8,8 +8,8 @@ const func = async function (hre) {
     // Deploy our own Finder
     // const Finder = await deployments.get("Finder");
 
-    // Using Finder deployed by UMA on goerli
-    const Finder = { address: "0xE60dBa66B85E10E7Fd18a67a6859E241A243950e" };
+    // Using Finder deployed by UMA on mainnet
+    const Finder = { address: "0x40f941E48A552bF496B154Af6bf55725f18D77c3" };
```

Comment:
Test case is failing due to changed finder address

Reply:
nice solution! I don't think we need this right now, we are only supporting main net for now.